src/lib/hooks/useGroups.test.ts

```diff
@@ -0,0 +1,36 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useGroups from './useGroups';
+import firebase from 'firebase';
+
+const mockFirebaseConfig = {
+  apiKey: 'mockApiKey',
+  authDomain: 'mockAuthDomain',
+  databaseURL: 'mockDatabaseURL',
+  projectId: 'mockProjectId',
+  storageBucket: 'mockStorageBucket',
+  messagingSenderId: 'mockMessageSenderId',
+  appId: 'mockAppId',
+  measurementId: 'mockMeasurementId',
+};
+
+describe('useGroups', () => {
+  beforeAll(async () => {
+    firebase.initializeApp(mockFirebaseConfig);
+    await firebase.firestore().enableNetwork();
```

Comment:
Question: what does this do?

Reply:
Jest was staying open due to the firestore continuing to listen for requests: https://github.com/facebook/jest/issues/7287#issuecomment-488886582. This ensures that it doesn't do so when testing

Reply:
I can add the url as a coment

Reply:
Yes please! I wouldn't have realized that if it wasn't for the explanation + url